assets/app/features/upgrade/i18n/en.json

```diff
@@ -32,50 +32,50 @@
-                "title": "Create a backup of the Administration Server configuration and download it in your browser.",
-                "description": "Keep this backup somewhere safe in case of emergency. With this backup and a fresh deployment of SUSE OpenStack Cloud 6 your cloud can be returned to it’s previous state.",
+                "title": "Create a backup of the Administration Server configuration and download it in your browser",
+                "description": "This backup contains information about the cloud deployment and configuration. Keep this backup somewhere safe in case of emergency.",
                 "form": {
                     "backup": "Download Backup of Administration Server"
                 }
             },
             "admin-repository-checks": {
-                "name": "",
-                "title": "Upgrading to SUSE OpenStack Cloud 7 requires software repositories for SLES12 SP2 and SUSE OpenStack Cloud 7.",
-                "description": "Update the URLs of the following software repositories and click 'Check' to verify the repositories are available. When all repositories have been verified click on 'Next' to proceed.",
+                "title": "Upgrading to SUSE OpenStack Cloud 7 requires updated software repositories for the Administration Server",
+                "description": "Prepare the software repositories locally or via SMT and update the URLs in the Administration Server configuration correspondingly. Click \"Check\" to verify all repositories are available. When all repositories have been verified click on \"Next\" to proceed.",
                 "form": {
                     "check-again": "Check"
                 }
             },
             "upgrade-admin": {
-                "title": "Upgrade the Administration Server",
-                "description": "The button 'Upgrade Administration Server' begins the update and upgrade process of the administration server. The SLES 12 SP1 operating system of the administration server will be upgraded to SLES 12 SP2. This might take up to half an hour so please be patient.",
+                "title": "Upgrade the Administration Server to SUSE OpenStack Cloud 7",
+                "description": "Please wait until the Administration Server has been upgraded and rebooted.",
+                "notice-title": "IMPORTANT!",
+                "notice-body": "This process might take quite some time. Please do not close the browser window or refresh the page.",
                 "form": {
                     "upgrade-admin": "Upgrade Administration Server"
                 }
             },
             "database-configuration": {
-                "name": "",
-                "title": "Enter Credentials and/or Information to Create or Connect to an OpenStack PostgreSQL Server for the OpenStack Database Migration.",
+                "title": "Enter Credentials and/or Information to create or connect to a PostgreSQL database for the Crowbar database",
```

Comment:
That sentence looks weird. What about *...to a PostgreSQL database that should serve as the Crowbar database* ?